core/cap-0034.md

```diff
@@ -49,6 +52,80 @@ dependent transactions would then perform operations which the smart contract
 had intended to be performed only if `T` had been executed. This would
 potentially be arbitrarily bad for the smart contract.
 
+So here are the conditions under which the above race occurs, and the semantics
+of the current and proposed protocols differ:
+
+1. A smart-contract client submits a series of transactions `T1`, `T2` with
+`T2`'s sequence number one greater than `T1`'s.  `T2` is intended to execute
+if and only if `T1` executes; the smart contract is depending upon `T2` having
+the wrong sequence number if `T1` is not applied (and therefore does not consume
+a sequence number).  The smart contract sets an expiration time on `T1` (as our
+[documentation](https://www.stellar.org/developers/guides/concepts/transactions.html)
+"highly advises" that clients do).
+
+2. The ledger happens first to have capacity to accept `T1` into a transaction
+set between `T1`'s expiration time and the time at which it will turn out (of
+course the network can not foresee that this will eventually be the case) that
+the next ledger will close.
+
+3. `T1` happens at that point to be in the transaction queue of the validator
+who (though again this is not predictable yet) will turn out to win nomination.
+That node builds a transaction set that contains `T1`.
+
+4. The transaction set containing `T1` wins nomination, but (at least) one of
+the candidate `closeTime`s is greater than `T1`'s expiration time.
+
+5. Transactions are applied using the maximum of all candidate `closeTime`s as
+the new last ledger close time.  `T1` therefore returns `txTOO_LATE` and is
+not applied.  However, as it reached ledger close before being failed, it is
+charged a fee and consumes a sequence number.
+
+6. `T2` gets into an externalized transaction set (possibly, but not
+necessarily, the same one that `T1` got into) before it too expires.  It
+has the correct sequence number, because `T1` consumed a sequence number.  It
+is therefore applied -- even though its preconditions were intended to include
+guarantees that `T1` had made by executing, so the postconditions of `T2`
+could violate any of the smart contract's intended invariants which had
+depended upon `T1` having executed before `T2` could so so.
+
+With the behavior proposed in this CAP, step #3 -- a node building a transaction
```

Comment:
This paragraph reads strangely because you haven't actually described the behavior yet. Probably best to think of "Motivation" as "What's the problem?", "Specification" as "Here's my proposed behavior", and "Design Rationale" as "Why is my proposed behavior the best way to solve the problem?"

Reply:
As described in a comment above, I did move that down to the "Rationale" (along with other changes), by which point I have described the behavior (in the "Specification").